src/app/pages/questions/services/audio-record.service.ts

```diff
@@ -11,24 +11,59 @@ import {
 
 import { DefaultAudioRecordOptions } from '../../../../assets/data/defaultConfig'
 import { LogService } from '../../../core/services/misc/log.service'
+import { RemoteConfigService } from 'src/app/core/services/config/remote-config.service'
+import { ConfigKeys } from 'src/app/shared/enums/config'
 
 @Injectable({
   providedIn: 'root'
 })
 export class AudioRecordService {
+  private samplingRate: number = DefaultAudioRecordOptions.SampleRate
+  private bitRate: number = DefaultAudioRecordOptions.BitRate
+  private encoder: string = DefaultAudioRecordOptions.AudioEncoder
+
   isRecording: boolean
   encoding = 'base64'
   data: any
 
   constructor(
-    private logger: LogService
-  ) {}
+    private logger: LogService,
+    private remoteConfig: RemoteConfigService
+  ) {
+    this.remoteConfig
+      .subject()
+      .subscribe(config => {
+        config
+          .getOrDefault(
+            ConfigKeys.AUDIO_SAMPLING_RATE,
+            String(this.samplingRate)
+          )
+          .then(rate => (this.samplingRate = Number(rate)))
+        config
+          .getOrDefault(
+            ConfigKeys.AUDIO_BIT_RATE,
+            String(this.bitRate)
+          )
+          .then(rate => (this.bitRate = Number(rate)))
+        config
+          .getOrDefault(
+            ConfigKeys.AUDIO_ENCODER,
+            this.encoder
+          )
+          .then(encoder => (this.encoder = encoder))
```

Comment:
i think it would be good to document these parameters and what values are supported.

Reply:
Yup added to the README.